requirements.txt

```diff
@@ -1,88 +1,52 @@
-appnope==0.1.0
-argh==0.26.2
-astroid==1.6.5
 atomicwrites==1.1.5
 attrs==18.1.0
-autopep8==1.3.5
-backcall==0.1.0
-black==19.10b0
 blinker==1.4
+black==19.10b0
 certifi==2018.4.16
 chardet==3.0.4
 click==6.7
 clickhouse-driver==0.1.1
-colorama==0.3.9
-configparser==3.5.0
 confluent-kafka==1.2.0
-contextlib2==0.5.5
 coverage==4.5.1
 datadog==0.21.0
-decorator==4.3.0
 deprecation==2.0.3
 docopt==0.6.2
-enum34==1.1.6
 flake8==3.7.8
 Flask==1.0.2
-funcsigs==1.0.2
 future==0.16.0
 honcho==1.0.1
 idna==2.7
-ipdb==0.11
-ipython==6.4.0
-ipython-genutils==0.2.0
-isodate==0.6.0
-isort==4.3.4
 itsdangerous==0.24
-jedi==0.12.0
 Jinja2==2.10.1
 jsonschema==3.0.1
-lazy-object-proxy==1.3.1
-linecache2==1.0.0
 lz4==2.0.0
 Markdown==2.6.11
 MarkupSafe==1.1.1
-mccabe==0.6.1
 mywsgi==1.0.3
 more-itertools==4.2.0
 mypy>=0.761
 packaging==17.1
 parsimonious==0.8.1
-parso==0.2.1
-pathlib2==2.3.2
-pathtools==0.1.2
-pbr==4.0.4
-petname==2.2
-pexpect==4.6.0
-pickleshare==0.7.4
-pluggy==0.13.0
-prompt-toolkit==1.0.15
-ptyprocess==0.5.2
 py==1.5.3
-Pygments==2.2.0
-pylint==1.9.2
-pycodestyle<2.6.0,>=2.5.0
 pyparsing==2.2.0
 pytest==5.2.4
 pytest-cov==2.5.1
-pytest-forked==1.1.3
 pytest-watch==4.2.0
-pytest-xdist==1.30.0
+colorama==0.3.9
```

Comment:
This list seems mostly alphabetically ordered, is there a reason to keep this out of order?

Reply:
Nope, but I can just reorder for aesthetics.